### ISSUE TITLE
Importing cluster fails too early to track deployment

### ISSUE THREAD

**Issue by andrew-morris-rgs** [rancher/turtles#861]
### What steps did you take and what happened?

Deploying a capi cluster with the CAPV and CAPR, deploying failing (not what I am trying to solve in this bug request), because of this the cluster never imported.

### What did you expect to happen?

To see the cluster object as soon as it is created, then see the machine objects as the soon the begin to provision.

### Rancher Turtles version

0.13.0

### Label(s) to be applied

/kind bug

**Comment by alexander-demicev** [comment]
can I ask you to provide more details? The cluster is imported in Rancher only if it was successfully created with CAPI

**Comment by andrew-morris-rgs** [comment]
> can I ask you to provide more details? The cluster is imported in Rancher only if it was successfully created with CAPI

Yep. That if is the issue I am talking about. I would expect turtles to track a capi clusters whole lifecycle, including if it fails at some point during creation. Currently that is pretty opaque.

**Comment by alexander-demicev** [comment]
CAPI cluster status will be available in the Turtles UI extension. Closing this issue since it's related to UI

**Comment by andrew-morris-rgs** [comment]
I assume this is project referenced: https://github.com/rancher/capi-ui-extension

I will check it out!
